edx_sga/tasks.py

```diff
@@ -143,11 +202,19 @@ def zip_student_submissions(course_id, block_id, locator_unicode, username):
     """
     locator = BlockUsageLocator.from_string(locator_unicode)
     submissions_dir_name = _get_submissions_base_name(username, block_id, course_id)
-    destination_path = _get_dir_path(submissions_dir_name, locator)
-    _remove_existing_artifacts(destination_path)
-    os.mkdir(destination_path)
-    _collect_student_submissions(block_id, course_id, locator, destination_path)
-    _compress_folder(destination_path)
+    destination_path = _get_abs_local_path(submissions_dir_name, locator)
```

Comment:
This line runs for either storage type. That means we would still be using an absolute local path (including the `DATA_DIR` path, which defaults to `/edx/app/edxapp/data`) if S3 storage is being used. Is this correct/intentional/necessary?